example/lambdas/python-processing/package.json

```diff
@@ -10,10 +10,12 @@
   },
   "scripts": {
     "test": "python -m unittest",
-    "python-lint": "pylint *.py",
+    "python-lint": "pipenv run pylint *.py",
     "lint": "npm run python-lint",
-    "build": "pip install -r requirements-dev.txt && pip install -r requirements.txt",
-    "package": "npm run build",
+    "clean": "rm -rf .venv",
+    "build": "pip install pipenv && PIPENV_VENV_IN_PROJECT=1 pipenv install --dev --deploy --ignore-pipfile",
```

Comment:
Good news: this works swimmingly in conda.

Reply:
Glad to hear!